Description: unbreak install foreach and do individual installation
 Long command line would break the execution.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Forwarded: no
Last-Update: 2020-05-24

---

--- grpc-1.22.0.orig/Makefile
+++ grpc-1.22.0/Makefile
@@ -2980,13 +2980,17 @@ install-headers: install-headers_c insta
 
 install-headers_c:
 	$(E) "[INSTALL] Installing public C headers"
-	$(Q) $(foreach h, $(PUBLIC_HEADERS_C), $(INSTALL) -d $(prefix)/$(dir $(h)) && ) exit 0 || exit 1
-	$(Q) $(foreach h, $(PUBLIC_HEADERS_C), $(INSTALL) $(h) $(prefix)/$(h) && ) exit 0 || exit 1
+	for h in $(PUBLIC_HEADERS_C); do \
+		$(INSTALL) -d $(prefix)/$$(dirname $$h); \
+		$(INSTALL) $$h $(prefix)/$$h; \
+	done
 
 install-headers_cxx:
 	$(E) "[INSTALL] Installing public C++ headers"
-	$(Q) $(foreach h, $(PUBLIC_HEADERS_CXX), $(INSTALL) -d $(prefix)/$(dir $(h)) && ) exit 0 || exit 1
-	$(Q) $(foreach h, $(PUBLIC_HEADERS_CXX), $(INSTALL) $(h) $(prefix)/$(h) && ) exit 0 || exit 1
+	for h in $(PUBLIC_HEADERS_CXX); do \
+		$(INSTALL) -d $(prefix)/$$(dirname $$h); \
+		$(INSTALL) $$h $(prefix)/$$h; \
+	done
 
 install-static: install-static_c install-static_cxx
 
